### PR TITLE
Introduce arbiter

### PR DESCRIPTION
**Pull request by truongnmt**
Introduces Arbiter:
- Get Job
	- `Jid int`
	- `Command string`
	- `Priority int`
- Push Job ID and Job Priority to queue named: `queued-list`
	- `ZJobToQueue(util.JobQueueKey, job.Jid, job.Priority)`
	- Use ZAdd, add to sorted set with priority, smaller is more priority
	- https://redis.io/commands/zadd
- Push Job command to hash:
	- `HSet(util.JobInfoPrefix+strconv.Itoa(job.Jid), util.JobCmdKey, job.Command)`
	- Store hash with key is `job-jobid`, value is `job.Command`, to field `util.JobCmdKey`
	- https://redis.io/commands/hset

### CODE REVIEW

File: arbiter/arbiter.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+func Enqueue(eIns entity.Entity, job Job) (int, error) {
+	err := eIns.Redis.ZJobToQueue(util.JobQueueKey, job.Jid, job.Priority)
+	if err != nil {
+		logger.Error("error when add job to queue", err)
+		return 0, err
+	}
+
+	err = eIns.Redis.HSet(util.JobInfoPrefix+strconv.Itoa(job.Jid), util.JobCmdKey, job.Command)
+	if err != nil {
+		logger.Error("error when add job command")
```
Comment (marverlous811) [inline arbiter/arbiter.go:27]: Xem có cần xóa đi job khỏi queue nếu không add được command vào không ?

File: test/enqueue_test.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	"xqueue/entity"
+	"xqueue/util"
+)
+
+func TestEnqueue(t *testing.T) {
+	assert := assert.New(t)
+
+	eIns := entity.CreateNewEntity()
+	defer eIns.Release()
+
```
Comment (marverlous811) [inline test/enqueue_test.go:22]: Đặt 1 cái defer để xóa dữ liệu khi test đi

File: arbiter/arbiter.go (hunk 1 above)
Comment (truongnmt) [reply arbiter/arbiter.go:27]: done: f234b5b

File: test/enqueue_test.go (hunk 2 above)
Comment (truongnmt) [reply test/enqueue_test.go:22]: Here u go: cf10948

**Pull request merged**